core/include/detray/propagator/actors/parameter_transporter.hpp

```diff
@@ -63,8 +63,12 @@ struct parameter_transporter : actor {
             const free_matrix_t correction_term =
                 matrix::identity<free_matrix_t>() + path_correction;
 
-            return free_to_bound_jacobian * correction_term *
-                   stepping.transport_jacobian() * bound_to_free_jacobian;
+            algebra::generic::math::set_inplace_product_right(
```

Comment:
Shouldn't it be `set_inplace_product_left` if this function is supposed to change `free_to_bound_jacobian`?